service/security/shiro/src/main/java/org/eclipse/kapua/service/authentication/shiro/CredentialsFactoryImpl.java

```diff
@@ -39,8 +39,8 @@ public ApiKeyCredentials newApiKeyCredentials(String apiKey) {
     }
 
     @Override
-    public JwtCredentials newJwtCredentials(String jwt) {
-        return new JwtCredentialsImpl(jwt);
+    public JwtCredentials newJwtCredentials(String jwt, String idToken) {
```

Comment:
Hey @gbarbon,
can you change the header of this file to " * Copyright (c) 2011, 2020 Eurotech and/or its affiliates and others" ? 
Because last change is from the current year 😄